### PR TITLE
feat: Add isTextMessageEncrypted to notify_verb_builder.dart

### PR DESCRIPTION
**Pull request by sitaram-kalluri**
**- What I did**
1. Add isTextMessageEncrypted to notify verb builder to support encrypting the notification text message.

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Review by gkc — CHANGES_REQUESTED** [review]
This PR doesn't stand alone, it's one part of the larger effort to add encryption support. Given that, please can you add details about all the changes into the main github issue for this feature, and provide a link in this PR description to the main github issue

**Comment by sitaram-kalluri** [comment]
The following are the PR's in at_client and at_server related to encrypting the notify text

at_client: [PR-611](https://github.com/atsign-foundation/at_client_sdk/pull/611)
at_server: [PR-812](https://github.com/atsign-foundation/at_server/pull/812)

File: at_commons/lib/src/verb/notify_verb_builder.dart [hunk 1]
```diff
@@ -93,6 +97,9 @@ class NotifyVerbBuilder implements VerbBuilder {
       ccd ??= false;
       command += 'ttr:$ttr:ccd:$ccd:';
     }
+    if(isTextMessageEncrypted){
```
Comment (murali-shris) [inline at_commons/lib/src/verb/notify_verb_builder.dart]: please dart format this file

**Review by gkc — APPROVED** [review]

File: at_commons/lib/src/verb/notify_verb_builder.dart (hunk 1 above)
Comment (sitaram-kalluri) [reply at_commons/lib/src/verb/notify_verb_builder.dart]: Ran dart formatter and pushed the changes.

**Pull request merged**